agasc/tests/test_multi_agasc_file.py

```diff
@@ -0,0 +1,40 @@
+import os
+import tables
+from ska_path import ska_path
+from astropy.table import Table
+import tempfile
+
+from .. import agasc
+
+DATA_ROOT = ska_path('data', 'agasc')
+tables_open_file = getattr(tables, 'open_file', None) or tables.openFile
+
+def test_multi_agasc():
+
+    tempdir = tempfile.mkdtemp()
+
+    # Make two custom agasc files from the miniagasc, using 20 stars from
```

Comment:
I suppose that switching "back an forth" would be more explicit if I did a get_cone_stars from one then the other, and then did the final star position check from the first one again instead of from the second one.